pkg/term/term_windows.go

```diff
@@ -1,12 +1,16 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	"github.com/docker/docker/pkg/term/windows"
+	"github.com/docker/docker/vendor/src/github.com/Azure/go-ansiterm/winterm"
```

Comment:
@gabrielhartmann the `github.com/Sirupsen/logrus` above is vendored as well. as @cpuguy83 said, the path of vendored packages is added to GOPATH while building the docker binary and thus we don't need to have the full pkg path.

For windows CLI , the GOPATH must look like: `GOPATH=/c/gopath:/c/gopath/src/github.com/docker/docker/vendor` ([you can find more here](https://ahmetalpbalkan.com/blog/compiling-docker-cli-on-windows/))